### PR TITLE
Feature/master_TRACTION-362

### PR DESCRIPTION
**Pull request by rishabh-trac**
# Critical Changes

# Changes
StaffAvailabilityController.cls
staffAvailability, staffAvailabilityItem, lookup lwc comps

# Issues Closed
TRACTION-362

### CODE REVIEW

File: force-app/main/default/lwc/staffAvailability/staffAvailability.js [hunk 1]
```diff
@@ -61,7 +61,6 @@ export default class StaffAvailability extends LightningElement {
 
 	renderedCallback() {
 		console.log('RECORD ID: ', this.recordId);
```
Comment (ghost) [inline force-app/main/default/lwc/staffAvailability/staffAvailability.js:63]: remove console.logs

File: force-app/main/default/lwc/staffAvailability/staffAvailability.js [hunk 2]
```diff
@@ -90,7 +89,7 @@ export default class StaffAvailability extends LightningElement {
 			console.log('ASSIGNMENT DATA', result);
```
Comment (ghost) [inline force-app/main/default/lwc/staffAvailability/staffAvailability.js:89]: remove console.logs

File: force-app/main/default/lwc/staffAvailabilityItem/staffAvailabilityItem.js [hunk 3]
```diff
@@ -144,9 +147,8 @@ export default class StaffAvailabilityItem extends LightningElement {
 			this.validateLookup();
 			if (this.lookupConfig.lookupErrors.length > 0) return;
 
-			const selection = this.template.querySelector('c-lookup').getSelection();
-			console.log('LOOKUP SELECTION: ', JSON.parse(JSON.stringify(selection)));
-			recordPayload[this.namespace+'Care_Facility__c'] = selection[0].id;
+			console.log('LOOKUP SELECTION: ', JSON.parse(JSON.stringify(this.selectedRecord)));
```
Comment (ghost) [inline force-app/main/default/lwc/staffAvailabilityItem/staffAvailabilityItem.js:150]: remove console.log

File: unpackaged/config/community/experiences/Traction_Thrive1/views/customAboutUs.json [hunk 4]
```diff
@@ -44,7 +44,7 @@
...
...ree licenses for this initiative. Excited by the potential to help more people, the united team (Thrive, Traction on Demand and Salesforce) began to develop a plan that could scale the impact of this initiative beyond BC or Canada, to global proportion. Knowing that innovation and speed were the only hope to scale in time and that any friction to the platform’s distribution, adaptability or quality would slow it’s adoption, the team together made the choice to move to an open source model. Salesforce brought on their opensource expert team, the same team that built and scaled the Non-Profit Success Platform to hundreds of thousands of NGO’s, to help apply the same scale to the platform.</span></p><p class=\"ql-quote-block\"><br></p><p class=\"ql-quote-block\"><span style=\"font-size: 16px; font-family: helvetica;\">Our goal, all of ours, is to try to help save lives, to preserve the integrity of the healthcare systems and to support our invaluable healthcare workers. We applaud the tireless dedication and courage of all of our health care workers around the globe. It is our sincere hope that this product can make their jobs a little bit better.</span></p></blockquote><p>&nbsp;</p>"
```
Comment (ghost) [inline unpackaged/config/community/experiences/Traction_Thrive1/views/customAboutUs.json:47]: have these about us links been tested?  there was an issue with them previously that has been fixed.

File: force-app/main/default/lwc/staffAvailability/staffAvailability.js (hunk 1 above)
Comment (rishabh-trac) [reply force-app/main/default/lwc/staffAvailability/staffAvailability.js:63]: actually, there are many console logs in these lwc components
I thought of not removing these as they might be left by developers intentionally.
I can do a commit of removing all console logs from all related lwc comps if you agree?

File: force-app/main/default/lwc/staffAvailability/staffAvailability.js (hunk 2 above)
Comment (rishabh-trac) [reply force-app/main/default/lwc/staffAvailability/staffAvailability.js:89]: same as the above comment/reply!

File: force-app/main/default/lwc/staffAvailabilityItem/staffAvailabilityItem.js (hunk 3 above)
Comment (rishabh-trac) [reply force-app/main/default/lwc/staffAvailabilityItem/staffAvailabilityItem.js:150]: same as the above comment/reply!

File: unpackaged/config/community/experiences/Traction_Thrive1/views/customAboutUs.json (hunk 4 above)
Comment (rishabh-trac) [reply unpackaged/config/community/experiences/Traction_Thrive1/views/customAboutUs.json:47]: yup, i tested with master branch, images were not displaying
it my mistake of using static src=\"https://sandbox-power-speed-9033-dev-ed-1717d2f45a2.cs45.force.com/TractionThrive/file-asset/LogoTHToDSmall?v=1\"
so the update contains updated url for images and is correct now
and we have to go with latest commit changes.

File: force-app/main/default/classes/StaffAvailabilityController.cls [hunk 5]
```diff
@@ -117,15 +117,12 @@ public with sharing class StaffAvailabilityController {
...
 	public static Assignment__c save(Assignment__c assignment) {
 
-		SecureDml upsertObject = new SecureDml();
-		try {
-			upsertObject.dmlUpsert(Assignment__c.SObjectType, new List<Assignment__c> {assignment}, Assignment__c.Id);
-		}
-		catch(Exception e) {
-			throw new AuraHandledException(e.getMessage() + ' : RECORD -> ' + JSON.serializePretty(assignment));
+		Database.UpsertResult saveResult = Database.upsert(assignment, false);
+		if (!saveResult.isSuccess()) {
```
Comment (cwico) [inline force-app/main/default/classes/StaffAvailabilityController.cls:121]: Nice!

**Review by cwico — APPROVED** [review]

**Pull request merged**